tests/test_admin/test_view.py

```diff
@@ -1479,6 +1608,44 @@ def test_post_link_and_enroll_into_program(
 
         self._assert_django_messages(response, expected_messages)
 
+    @mock.patch("enterprise.models.CourseCatalogApiClient")
+    @mock.patch("enterprise.api_client.ecommerce.ecommerce_api_client")
+    @mock.patch("enterprise.admin.views.EnrollmentApiClient")
+    @mock.patch("enterprise.admin.forms.EnrollmentApiClient")
+    def test_order_failure_message(
```

Comment:
May be rename this test to `test_order_failure_message_bulk` or something. Also if we can move it below the `test_order_failure_message`, would be great because both are covering same cases.